Mail/Views/Thread/WebView.swift

```diff
@@ -63,7 +63,7 @@ struct WebView: UIViewRepresentable {
         ) {
             if let url = navigationAction.request.url, Constants.isMailTo(url) {
                 decisionHandler(.cancel)
-                (parent.window?.windowScene?.delegate as? SceneDelegate)?.handleUrlOpen(url)
+                parent.openUrl.callAsFunction(url)
```

Comment:
Is it possible to write parent.openUrl(url)?
Just like the environment property dismiss

Reply:
It is